go/libraries/doltcore/sqle/tables.go

```diff
@@ -224,16 +224,14 @@ func (t *AlterableDoltTable) AddColumn(ctx *sql.Context, column *sql.Column, ord
 		return err
 	}
 
-	sch, err := table.GetSchema(ctx)
-	if err != nil {
-		return err
-	}
-
 	tag := extractTag(column)
 	if tag == schema.InvalidTag {
-		tag = schema.AutoGenerateTag(sch)
+		// TODO: are we sure we want to silently autogen if we fail to parse?
```

Comment:
Yes, because a comment that doesn't contain a tag is still legitimate SQL. We just treat it as if there were not comment on the column.

(We don't actually store the comment in dolt, but someday maybe)